package.json

```diff
@@ -42,6 +42,7 @@
     "bignumber.js": "~2.4.0",
     "bowser": "~1.5.0",
     "classnames": "~2.2.3",
+    "clipboard-js": "^0.3.1",
```

Comment:
We could instead add a package.json for the demo site and have it abit more independent from the reset